app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -166,7 +166,7 @@ def create
 
     logger.debug("Creating meeting #{params[:meetingID]} on BigBlueButton server #{server.id}")
     # Pass along all params except the built in rails ones
-    uri = encode_bbb_uri('create', server.url, server.secret, pass_through_params)
+    uri = encode_bbb_uri('create', server.url, server.secret, pass_through_params('create'))
```

Comment:
Pass the list itself, not the name.
```suggestion
    uri = encode_bbb_uri(
      'create',
      server.url,
      server.secret,
      pass_through_params(Rails.configuration.x.create_exclude_params)
    )
```